code/model/modelanimation_segments.cpp

```diff
@@ -1114,18 +1106,14 @@ namespace animation {
 		tl::optional<vec3d> offset, velocity, acceleration;
 		tl::optional<float> time;
 		CoordinateSystem coordSystem = CoordinateSystem::COORDS_PARENT;
-		bool isAbsolute = false;
+		ModelAnimationCoordinateRelation isAbsolute = ModelAnimationCoordinateRelation::RELATIVE_COORDS;
 
 		if (optional_string("+Vector:")) {
 			vec3d parse;
 			stuff_vec3d(&parse);
 			offset = std::move(parse);
-			isAbsolute = optional_string("+Absolute");
-			bool relative = optional_string("+Relative");
-
-			if (isAbsolute && relative) {
-				error_display(1, "Translation cannot both be absolute and relative!");
-			}
+			int result = optional_string_one_of(3, "+Relative", "+Local", "+Absolute");
+			isAbsolute = static_cast<ModelAnimationCoordinateRelation>(result < 0 ? 0 : result);
```

Comment:
Considering how this appears a few times, I'd suggest a utility method to parse the values.